pkgs/tools/networking/ntpd-rs/default.nix

```diff
@@ -5,36 +5,30 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "ntpd-rs";
-  version = "0.3.7";
+  version = "1.0.0";
 
   src = fetchFromGitHub {
     owner = "pendulum-project";
     repo = "ntpd-rs";
     rev = "v${version}";
-    hash = "sha256-AUCzsveG9U+KxYO/4LGmyCPkR+w9pGDA/vTzMAGiVuI=";
+    hash = "sha256-pR0mswnPII7w1MtOKnG98rl2GRx1ojntfaJjkJJ4Zsw=";
   };
 
-  cargoHash = "sha256-6FUVkr3uock43ZBHuMEVIZ5F8Oh8wMifh2EokMWv4hU=";
+  cargoHash = "sha256-kYPIFOCWTrn9zayIaYGSESFIM7r7N+laf8bYQVCUKRc=";
 
   checkFlags = [
     # doesn't find the testca
     "--skip=keyexchange::tests::key_exchange_roundtrip"
-    # seems flaky
+    # seems flaky?
```

Comment:
`?` indeed. What's the confusion?

Reply:
Since the new version it seems to always fail.